e2e/nomostest/reset.go

```diff
@@ -282,7 +282,7 @@ func ResetRepoSyncs(nt *NT, rsList []v1beta1.RepoSync) error {
 		nn := client.ObjectKeyFromObject(obj)
 		nt.T.Logf("[RESET] Waiting for deletion of RepoSync %s ...", nn)
 		tg.Go(func() error {
-			return WatchForNotFound(nt, kinds.RepoSyncV1Beta1(), obj.Name, obj.Namespace)
+			return WatchForNotFound(nt, kinds.RepoSyncV1Beta1(), nn.Name, nn.Namespace)
```

Comment:
line 361 also needs to be updated.

Reply:
I'll update it for consistency, but technically it doesn't need it, because the WatchForNotFound is synchronous. The range item won't be overwritten until after it exits.

Reply:
Actually, I'll change that loop to use the TaskGroup too